pennylane_lightning/lightning_qubit/_state_vector.py

```diff
@@ -0,0 +1,356 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Class implementation for state-vector manipulation.
+"""
+
+try:
+    from pennylane_lightning.lightning_qubit_ops import (
+        allocate_aligned_array,
+        get_alignment,
+        best_alignment,
+        StateVectorC64,
+        StateVectorC128,
+    )
+except ImportError:
+    pass
+
+from itertools import product
+import numpy as np
+
+import pennylane as qml
```

Comment:
Not a blocker at all, but I'm wondering whether we should introduce `isort` in addition to `black` to take care of imports. This would automatically fix several `pylint` warnings and issues.

Reply:
Let me add this to lightning :)